core/data/src/main/java/uk/gov/gchq/gaffer/data/elementdefinition/view/View.java

```diff
@@ -143,6 +144,15 @@ public boolean hasEdgeFilters() {
                 || hasEdgeFilters(ViewElementDefinition::hasPreAggregationFilters);
     }
 
+    @JsonIgnore
+    public Map<String, ViewElementDefinition> getElements() {
```

Comment:
This is only used in one place. It might be better to put this where it is used, so we don't encourage others to use it without realising a new hash map is created with every call.

Reply:
Ok - fixed now.